util/pace/util/_legacy_restart.py

```diff
@@ -38,29 +42,43 @@ def open_restart(
     Returns:
         state: model state dictionary
     """
+
     if tracer_properties is None:
         restart_properties = RESTART_PROPERTIES
+        restart_properties.update(extra_restart_properties)  # add microphysics
+        # original restart properties don't have that stuff
     else:
         restart_properties = {**tracer_properties, **RESTART_PROPERTIES}
+        restart_properties.update(extra_restart_properties)
+
     rank = communicator.rank
     tile_index = communicator.partitioner.tile_index(rank)
     state = {}
     if communicator.tile.rank == constants.ROOT_RANK:
         for file in restart_files(dirname, tile_index, label):
+
             state.update(
                 load_partial_state_from_restart_file(
                     file, restart_properties, only_names=only_names
                 )
             )
+
         coupler_res_filename = get_coupler_res_filename(dirname, label)
         if filesystem.is_file(coupler_res_filename):
             if only_names is None or "time" in only_names:
                 with filesystem.open(coupler_res_filename, "r") as f:
                     state["time"] = io.get_current_date_from_coupler_res(f)
-    if to_state is None:
-        state = communicator.tile.scatter_state(state)
-    else:
-        state = communicator.tile.scatter_state(state, recv_state=to_state)
+
+    # if not fortran_restart:
+    #     if to_state is None:
+    #         state = communicator.tile.scatter_state(state)
+    #     else:
+    #         state = communicator.tile.scatter_state(state, recv_state=to_state)
+    # Ajda
+    # fortran restart files don't need to be scattered as each tile has its own
+    # file ... At least when it's run on 6 ranks.
+    # Not sure what happens if you have more (do you get more restart files?) ...
```

Comment:
Please restore this code. We need to be able to load restarts when we have more than 6 ranks.

```suggestion
    # if not fortran_restart:
    #     if to_state is None:
    #         state = communicator.tile.scatter_state(state)
    #     else:
    #         state = communicator.tile.scatter_state(state, recv_state=to_state)
    # Ajda
    # fortran restart files don't need to be scattered as each tile has its own
    # file ... At least when it's run on 6 ranks.
    # Not sure what happens if you have more (do you get more restart files?) ...
```